pkg/plugins/policies/meshratelimit/api/v1alpha1/deprecated.go

```diff
@@ -9,6 +11,35 @@ func (t *MeshRateLimitResource) Deprecations() []string {
 	var deprecations []string
 	if len(pointer.Deref(t.Spec.From)) > 0 {
 		deprecations = append(deprecations, "'from' field is deprecated, use 'rules' instead")
+		for i, rule := range pointer.Deref(t.Spec.From) {
+			if rule.Default.Local != nil && isStatusInvalid(pointer.Deref(rule.Default.Local)) {
+				deprecations = append(deprecations, fmt.Sprintf("'spec.from[%d].default.local.http.requestRate.status' must be 400 or higher. Please update your configuration.", i))
```

Comment:
should we mention why that's the case? that envoy uses 429 when that happens?

Reply:
should this be in deprecations or in [warnings](https://github.com/kumahq/kuma/blob/62a33aec0311c7484df58f2374e5683f9a6ef93d/pkg/plugins/policies/meshpassthrough/plugin/v1alpha1/plugin.go#L42)?